portfolio/src/main/webapp/index.html

```diff
@@ -1,100 +1,129 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
     <script src="fetcher.js"></script>
   </head>
   <body class="body">
     <div id="content">
-        <div class="topBar">
-            <img src="images/mock pfp.jpg" class="pfp" alt="HTML5">
+      <div class="topBar">
+        <img src="images/mock pfp.jpg" class="pfp" alt="HTML5" />
 
-            <h1><strong>Josiah Paintsil's World</strong></h1>
+        <h1><strong>Josiah Paintsil's World</strong></h1>
 
-            <nav class="socialMedia">
-                <a href = "https://github.com/jpaints3">
-                    <img src="images/github icon.png" alt="HTML5">
-                </a>
-                <a href = "https://www.linkedin.com/in/josiahpaintsil/">
-                    <img src="images/linkedin icon.png" alt="HTML5">
-                </a>
-                <a href = "https://www.instagram.com/jpaints3">
-                    <img src="images/ig icon.png" alt="HTML5">
-                </a>
-            </nav>
-
-        </div>
-      
-
-        <div class="intro">
-            <p>Hi everyone, nice to meet you! 
-          My name is Josiah and I am a sophomore at Colgate. 
-          Big mad scientist energy! XD</p>
-        </div>
+        <nav class="socialMedia">
+          <a href="https://github.com/jpaints3">
+            <img src="images/github icon.png" alt="HTML5" />
+          </a>
+          <a href="https://www.linkedin.com/in/josiahpaintsil/">
+            <img src="images/linkedin icon.png" alt="HTML5" />
+          </a>
+          <a href="https://www.instagram.com/jpaints3">
+            <img src="images/ig icon.png" alt="HTML5" />
```

Comment:
Aside: Do these src values work? I'm surprised you can have a space in the `src` 🤔